geoips/commandline/geoips_command.py

```diff
@@ -328,11 +345,22 @@ def _get_cli_logger(self):
             default="interactive",
             choices=["interactive", "debug", "info", "warning", "error"],
         )
+        independent_parser.add_argument(
+            "--warnings",
+            type=str,
+            default=os.getenv("GEOIPS_WARNING_LEVEL", "ignore"),
```

Comment:
Should this go in base_paths.py / future config ?  So we can better track what the various available settings / options are?

Reply:
Ah, probably. This was a last minute addition. I'll add this and will need reapproval.